hack/test-update-storage-objects.sh

```diff
@@ -37,6 +37,7 @@ KUBE_STORAGE_MEDIA_TYPE_PROTOBUF="application/vnd.kubernetes.protobuf"
 
 ETCD_HOST=${ETCD_HOST:-127.0.0.1}
 ETCD_PORT=${ETCD_PORT:-4001}
+ETCD_PREFIX=${ETCD_PREFIX:-randomPrefix}
```

Comment:
@lavalamp - my feeling is that this is enough (we are testing defaulg prefix "/registry" in all other suites and existing tests. If you really think that it's not enough, we can run this suite twice for different prefixes. But I think this is just waste of time.